translations/sp/messages.nl.yml

```diff
@@ -0,0 +1,22 @@
+saml.subject_name_id: Subject name id
+saml.request_id: RequestId
+saml.issuer: Uitgever
```

Comment:
Not sure if you should translate these SAML terms also goes for `authenticating_authority`